cfme/tests/webui/test_general_ui.py

```diff
@@ -339,6 +341,42 @@ def test_tls_openssl_verify_mode(temp_appliance_preconfig, request):
     assert appliance.advanced_settings["smtp"]["openssl_verify_mode"] == "none"
 
 
+@pytest.mark.ignore_stream("5.10")
+@pytest.mark.meta(blockers=[BZ(1741310)], automates=[1741310])
+@pytest.mark.parametrize("destination", ["AnsibleTowerExplorer", "ConfigManagement"])
+def test_add_provider_button_accordion(appliance, destination, has_no_providers):
+    """
+    Test that add_provider button is visible after clicking accordion on
+    Ansible Tower and Satellite Provider pages
+
+    Bugzilla:
+        1741310
+
+    Polarion:
+        assignee: pvala
+        casecomponent: WebUI
+        caseimportance: medium
+        initialEstimate: 1/30h
+        startsin: 5.11
+    """
+    if destination == "AnsibleTowerExplorer":
+        view = navigate_to(appliance.server, destination)
+        accordion_select = "job_templates"
+    else:
+        config_manager = ConfigManager()
+        view = navigate_to(config_manager, "All")
+        accordion_select = "configured_systems"
+
+    assert view.add_button.is_displayed
+    # now click somewhere on the accordion
+    getattr(view.sidebar, accordion_select).open()
+    # now click back at the providers page
+    view.sidebar.providers.open()
+    view.wait_displayed()
+    # assert that the button is still present
+    assert view.including_entities.add_button.is_displayed
```

Comment:
**Required:**
`view.including_entities` gives AttributeError. I think it should simply be `view.add_button.is_displayed`.

Reply:
Oh duh, I noticed this in local testing and though I pushed this fix -- I guess not!